benchmarks/src/main/scala/zio/redis/lists/LRangeBenchmarks.scala

```diff
@@ -0,0 +1,64 @@
+package zio.redis.lists
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+
+import zio.ZIO
+import zio.redis.{ BenchmarkRuntime, del, lRange, rPush }
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
+@Measurement(iterations = 15)
+@Warmup(iterations = 15)
+@Fork(2)
+class LRangeBenchmarks extends BenchmarkRuntime {
+  @Param(Array("400"))
```

Comment:
Bump to 500.

Reply:
Unfortunately the rediculous client hangs when the count is set to 500 for this command, however I could set it to 450.